cmd/registry/controller/resource-name.go

```diff
@@ -22,113 +22,113 @@ import (
 // Example:
 // projects/demo/locations/global/apis/petstore/versions/1.0.0/specs/openapi.yaml
 // projects/demo/locations/global/apis/-/versions/-/specs/-/artifacts/-
-type ResourceName interface {
-	GetArtifact() string
-	GetSpec() string
-	GetVersion() string
-	GetApi() string
-	String() string
-	GetParent() string
+type resourceName interface {
+	getArtifact() string
+	getSpec() string
+	getVersion() string
+	getApi() string
+	string() string
+	getParent() string
```

Comment:
I don't think we need to unexport these methods if we're already unexporting the entire struct. Particularly, we probably still want this type to implement [fmt.Stringer](https://go.dev/tour/methods/17).

Also a nitpick since these are being updated: [getters in Go don't usually include the word "get" in the method name](https://go.dev/doc/effective_go#Getters), so these can be renamed to `Artifact() string`, `Spec() string`, etc.

The same advice applies to the rest of this file and `resource.go`.

Reply:
Done.